### PR TITLE
update for spike 0.15.0

### PR DESCRIPTION
**Pull request by covingtondoan**
I tested and it works! 

I didn't prune my dependencies or `spike-page-id` as I think it's used in `layout.sgr`.

### Potential issues

1. I did a split to remove the `.sgr` extension on `post._path` links in the views. I don't know if there is a better way to get it to work. 
2. I used absolute paths for the CSS and JS in `_page_template.sgr` and `_post_template.sgr`. They weren't coming through otherwise.

### CODE REVIEW

File: root/app.js [hunk 1]
```diff
@@ -14,9 +12,9 @@ const collections = new Collections({
       files: 'posts/**',
       permalinks: Collections.jekyll.date,
       paginate: {
-        perPage: 3,
+        perPage: 2,
         template: 'views/_page_template.sgr',
-        output: (n) => `posts/page${n}.html`
+        output: (n) => `page${n}.html`
```
Comment (jescalan) [inline root/app.js:17]: why these changes here?

File: root/package.json [hunk 2]
```diff
@@ -8,22 +8,21 @@
...
+    "webpack": "^2.2.1"<% } %>
   },
   "devDependencies": {
     "ava": "^0.17.0",
     "rimraf": "^2.5.4",
-    "snazzy": "^5.0.0",
-    "spike-core": "^0.13.1",
-    "standard": "^8.0.0"
+    "snazzy": "^6.0.0",
+    "spike-core": "^0.13.3",
```
Comment (jescalan) [inline root/package.json]: this needs to be `next` as well

File: root/views/_page_template.sgr [hunk 3]
```diff
@@ -5,22 +5,22 @@ html
...
+      title My Cool Blog: Page {{ _currentPage.page }}
 
     block(name='stylesheets')
-      link(rel='stylesheet' href='css/index.css')
+      link(rel='stylesheet' href='/css/index.css')
 
   body(role='document')
     main(role='main' id='{{ pageId }}')
-      h1 Posts: Page {{ _currentPage.page }}
+      h1 My Cool Blog: Page {{ _currentPage.page }}
```
Comment (jescalan) [inline root/views/_page_template.sgr:19]: Why these text changes?

File: root/views/_page_template.sgr [hunk 4]
```diff
@@ -5,22 +5,22 @@ html
...
       ul
         each(loop='post in _currentPage.posts')
           li
-            a(href='{{ post._path }}') {{ post.title }}
+            a(href='{{ post._path.slice(0, -4) }}') {{ post.title }}
```
Comment (jescalan) [inline root/views/_page_template.sgr:23]: ?

**Review by jescalan — COMMENTED** [review]
Looking great so far, thank you for doing this! Just a couple comments 😁

File: root/views/_page_template.sgr (hunk 3 above)
Comment (covingtondoan) [reply root/views/_page_template.sgr:19]: Just to keep the top a little more consistent

File: root/views/_page_template.sgr (hunk 4 above)
Comment (covingtondoan) [reply root/views/_page_template.sgr:23]: For some reason, the URL pulls in the `.sgr` extension, so the slice removes it. Please tell me there is a better way.

File: root/app.js (hunk 1 above)
Comment (covingtondoan) [reply root/app.js:17]: 🤔 I think this is fine as is. No change necessary.

File: root/views/_page_template.sgr (hunk 4 above)
Comment (jescalan) [reply root/views/_page_template.sgr:23]: was this happening before, or only after the upgrade?

File: root/app.js (hunk 1 above)
Comment (jescalan) [reply root/app.js:17]: ok, maybe we can revert then?
